ton_client/src/tests/mod.rs

```diff
@@ -730,6 +743,14 @@ impl TestClient {
             .await
             .unwrap();
 
+        assert!(
+            matches!(
+                run_result.transaction["out_msgs"].as_array(),
+                Some(out_msgs) if !out_msgs.is_empty()
+            ),
+            "The giver's topup call should result in at least 1 internal outbound message"
+        );
+
```

Comment:
`assert` assumes test case check. You'd better use such error check here

```suggestion
        if run_result.transaction["out_msgs"][0].is_null() {
            panic!("The giver's topup call should result in at least 1 internal outbound message");
        }
```